Lib/sqlite3/__main__.py

```diff
@@ -62,7 +62,7 @@ def runsource(self, source, filename="<input>", symbol="single"):
         return False
 
 
-def main():
+def main(*argv):
```

Comment:
So you can call `main` with multiple positional arguments. Another option is to pass a sequence of arguments. If it helps, we could name it `*args`, since that is the norm.

Reply:
Changed to use the `*args` norm instead in 18eefdba8c3b2cef790d0562cf78a06ed1322411

Reply:
I did not have strong preference here. It just looked suspicious.

Reply:
Well, now it looks less suspicious :) I mark the conversation as resolved.